### ISSUE TITLE
Create supression mechanism

### ISSUE THREAD

**Issue by stephanenicolas** [groupon/dependency-injection-checks#10]
`@SuppressWarnings("name of the detector")` should suppress warnings related to a detector.

It's always useful to provide users with an escape door for cases where the users know what they are doing and unblock them.

**Comment by code-twister** [comment]
Good catch. This should be on the roadmap for next release. Thanks Stephane!
